Week1/SOLUTION_1.py

```diff
@@ -0,0 +1,16 @@
+#write a python script to enter any number to check it is prime or not
+#22bca26   19-06-2023
+num =int(input('enter number: '))
+# If given number is greater than 1
+if num > 1:
+	# Iterate from 2 to n / 2
+	for i in range(2, int(num/2)+1):

```

Comment:
please explain me this for statement.

Reply:
sorry , i copy this program from google. so give me time to explain you

Reply:
we have the first condition that the "num" variable has to be greater than 1.
then second started the "for" loop form "2" so the loop will start from 2 then and  (num/2)+1):, where "num" will be divided by 2 and increment by 1.